README.md

```diff
@@ -177,66 +179,66 @@ phpmyadmin:
+:point_right: _Keep in mind: Here we use a ready-made image for the container, which we do not modify afterwards. Therefore, we can fetch it with `image: phpmyadmin/phpmyadmin` and don’t need a separate dockerfile in the `docker/` folder, just like with our other containers._
 
-Docker-Container neustarten:
+Restart the Docker container:
 
     $ docker-compose up -d
 
-Danach ist phpMyAdmin erreichbar über:
+After that you can access phpMyAdmin in your browser:
 
     http://localhost:28080
 
 ---
 
-## Anleitung für Einsteiger\_innen :rocket:
+## Beginner’s Guide :rocket:
 
-### Worum geht es?
+### What is it all about?
 
-Virtualisierung! Docker lässt verschiedene __Anwendungen in Containern__ laufen, z. B. eine Datenbank, einen Webserver und in unserem Fall ein REDAXO dazu. Diese Container werden auf deinem System ausgeführt und benutzen dessen Ressourcen, laufen aber trotzdem vollständig isoliert. Anders als virtuelle Maschinen (VM), die komplette Betriebssysteme ausführen — wie z. B. Windows auf einem Mac, um Websites im Internet Explorer zu testen oder die Steuererklärung zu machen — sind Docker-Container sehr klein und performant! Man kann problemlos zahlreiche Container auf einem Sytem starten.
+Virtualization! Docker runs various __applications in containers__, eg. For example, a database, a web server and, in our case, a REDAXO. These containers run on your system and use its resources, but still run completely isolated. Unlike virtual machines (VMs) that run entire operating systems - such as: For example, Windows on a Mac to test websites in Internet Explorer or make tax returns - Docker containers are very small and performant! You can easily start many containers on a system.
 
-Wir benutzen Docker in diesem Projekt, um uns aus verschiedenen Containern eine __Entwicklungsumgebung__ _zusammenzustecken_, in der wir REDAXO betreiben können: Ein Container enthält die Datenbank, ein anderer den Apache-Webserver mit PHP. Deine lokale Entwicklungsumgebung, die du vorher auf deinem System eingerichtet hast — vielleicht auch mit Hilfe von Tools wie XAMPP (Windows) oder MAMP (Mac) — wird damit überflüssig, denn sie wird nun über Docker-Container abgebildet. Und das bringt viele Vorteile mit, von denen für uns erstmal nur diese relevant sind:
+We use Docker in this project to assemble a development environment in which we can run REDAXO by using different containers: one container contains the database, another the Apache web server with PHP. Your local development environment, which you have previously set up on your system - perhaps with the help of tools such as XAMPP (Windows) or MAMP (Mac) - is thus unnecessary, because it is now provided by Docker container. And that brings many advantages with it, of which only these are relevant for us:
```

Comment:
```suggestion
We use Docker in this project to build a dev environment from different containers in which we can run REDAXO: one container contains the database, another the Apache web server with PHP. Your local dev environment, which you have previously set up on your system—perhaps with the help of tools such as XAMPP (Windows) or MAMP (Mac)—is thus unnecessary, because it is now provided by Docker containers. And that brings a lot of advantages, of which only these are relevant for us:
```